lib/tlscontext.c

```diff
@@ -280,6 +280,7 @@ tls_context_setup_session(TLSContext *self)
   SSL *ssl;
   TLSSession *session;
   gint ssl_error;
+  long ssl_options;
```

Comment:
I just say only one: Manual page SSL_CTX_set_options(3ssl)
SYNOPSIS
        #include <openssl/ssl.h>

        long SSL_CTX_set_options(SSL_CTX *ctx, long options);

That internal variable is for prepare the value what SSL_CTX_set_options will get.
Is that satisfactory enough? ;-)